CONTRIBUTING.md

```diff
@@ -107,6 +110,12 @@ adjust the proof indentation to match any indentation changes made by /rewrap.
 Then, 'verify markup' will check that no lines became too long due to different
 indentation.  Finally, 'verify proof' is there because you might as well.
 
+In addition, you are encouraged to run `scripts/minimize label`, where label is
+the name of your theorem. This script attempts to shorten the proof using other
+theorems in the database. Over many proofs, this can make a big difference in
+speeding up various functions, such as loading the database into a tool.
+However, this step is not a requirement for submissions to be accepted.
```

Comment:
Perhaps "requirement" would be overstating it but we do, if I understand correctly, encourage this. The text in the linked thread which says "In the past we have occasionally run the entire [set.mm](http://set.mm/) through minimization, and that will probably happen again at some point in the future when there is a volunteer with the necessary CPU resources" has only become more true as set.mm has grown and the volunteer/CPU resources available for mass minimizations seem to have been even more scarce than when that was written.

```suggestion
We encourage doing this before submitting proofs.
```

Reply:
>  So we encourage (but do not require) that people minimize their proofs when it is not too much trouble to do so.

I tried to keep this consistent with the above policy as stated in the thread by mentioning both that it is encouraged (at the start of the paragraph) and that it is not required. That said, I'm fine with removing the last sentence so long as there's a consensus that we don't want it.

Reply:
Oh good point about the start of the paragraph already saying encouraged. I suppose I'd drop the last sentence and keep the rest but I guess it doesn't matter a whole lot to me.